files/mygui/openmw_spell_window.layout

```diff
@@ -15,13 +15,13 @@
 
         <Widget type="Widget" position="8 535 268 23" align="Left Bottom HStretch">
             <!-- Spell deletion button -->
-            <Widget type="Button" skin="MW_Button" align="Right Bottom" position="150 0 118 24" name="DeleteSpellButton">
+            <Widget type="Button" skin="MW_Button" align="Right Bottom" position="108 0 160 24" name="DeleteSpellButton">
```

Comment:
I did not mean that.
First of all, there is an AutoSizedButton widget, which adjusts its size depending on its label size.
Second, the sDeleteSpell is just too long for such button - it should be a quite small and non-intrusive.

Reply:
So use sDelete instead and use an autoresize box?

Reply:
I am afraid that it will not be so trivial.

Reply:
so what do you want me to do?

Reply:
1. Use the AutoSizedButton.
2. Use the sDelete instead of sDeleteSpell.
3. Adjust the FilterEdit width properly. More likely you will have to do it manually in the spellwindow.cpp.